docs/patch.md

```diff
@@ -53,7 +54,7 @@ The `set` method has the following signature:
 patch.set(path: string, value: JsonValue): Patch
 ```
 
-The return is the path instance itself to allow operation chaining.
+The return is the `Patch` instance itself to allow operation chaining.
```

Comment:
Sharing a curiosity since this was changed. This type of API where you can chain a sequence of methods together to apply them on the same object is called [Fluent Interface](https://en.wikipedia.org/wiki/Fluent_interface)

Reply:
Thanks for sharing!

Reply:
Pretty cool, thanks